a-kholmovaia/aika
# Ali/feat/frontend/auth screens

*Created authentication screen UI* 


 ![After](https://github.com/a-kholmovaia/aika/blob/ali/feat/frontend/auth_screens/frontend/docs/auth_screen.jpg)

## 🔧 Changes

- Implemented UI for authentication screen in the presentation layer
- Added username and password text fields
- Included a checkbox to differentiate between login and sign up

## 🚀 How to Test

1. Navigate to the authentication screen in the application.
2. Verify that the username and password text fields are present.
3. Check the checkbox functionality for toggling between login and sign up.
4. Ensure the UI meets the design specifications.

## 🔜 To Do

- Implement the client logic to send requests to the server.
- Route to the main menu if the authentication is successful.
- Display an error message to the user if the authentication fails.
